lib/config.js

```diff
@@ -83,7 +84,7 @@ class Config {
   /**
    * Get current config.
    * @param {string} key
-   * @param {*} val
+   * @param {*=} val
```

Comment:
??

Reply:
This changes for typings. Because this param is an optional